mautic/mautic
# Import owner by username

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | y
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | https://github.com/mautic/mautic/issues/2615
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
Just added support for owner username  for contact import

![image](https://user-images.githubusercontent.com/462477/30248999-80dae6d4-9632-11e7-8d4f-8772dac23704.png)

#### Steps to test this PR:
1.  Add column ownerusername to import
2. Set ownerusername at Match the columns from the imported file to Mautic's contact properties
3. Import and see of different owners were updated to contacts
